docs/user-guide/scripts-&-functions/overview.md

```diff
@@ -1,24 +1,22 @@
-# Python Coprocessor
+# Scripts & functions
 
-Python prevails in the data science and AI world. To avoid spending lots of time and effort transferring and transforming data, we provide the capability of executing Python scripts in the database. We call it the python coprocessor: co-processing for time series data analysis and processing.
+GreptimeDB supports running Python script inside the database. If the business logic is too complex to express via SQL, you can use python. Python prevails in the data science and AI world. To avoid spending lots of time and effort transferring and transforming data, we provide the capability of executing Python scripts in the database. We call it the python coprocessor: co-processing for time series data analysis and processing.
 We think the python coprocessor in GreptimeDB is a perfect replacement for stored procedures in traditional RDMS, and also the user can create SQL UDF(User Defined Function) by defining coprocessors.
 
-* [Hello, world](./hello.md)
-* [Coprocessor](./coprocessor.md)
-* [Vector](./vector.md)
-* [Input and Output](./io.md)
-* [Query Data](./query-data.md)
-* [Insert Data](./insert-data.md)
-* [Builtin Module](./builtins.md)
-* [Third-parties Libs](./third-parties.md)
-* [FAQ](./faq.md)
+* [Getting started](./getting-started.md)
+* [Function](./function.md)
```

Comment:
I want to move `data types` section before `functon`. The user must be familiar with data types to use these functions.